docs/features/marks.md

```diff
@@ -564,3 +566,137 @@ Plot.marks(
 ```
 
 A convenience method for composing a mark from a series of other marks. Returns an array of marks that implements the *mark*.plot function. See the [box mark](../marks/box.md) implementation for an example.
+
+## Custom marks
```

Comment:
```suggestion
## Rendering
```

If this section is about custom marks, it should also cover extending the Mark class and implementing other Mark methods such as *mark*.initialize. But that adds scope to this PR, so let’s keep it focused on rendering.

Also, this section describes both **render functions** (functional marks, shorthand for a custom mark) and **render transforms**. I think we should make clear that these are two different concepts even though they are similar.